client/src/stylesheets/components/_NewCampaign.scss

```diff
@@ -1,30 +1,75 @@
-.create_campaign_wrapper {
-	display: flex;
-	flex-direction: column;
-	align-items: center;
-	justify-content: center;
-}
+@import '../base/colors';
 
-.create_campaign_subroute_wrapper {
-	padding: 32px;
-	Width: 400px;
-}
+.create-campaign-breadcrumbs {
+  margin-top: 20px;
+  height: 40px;
+  text-align: center;
 
-.create_campaign_breadcrumbs {
-	width: 200px;
-	height: 40px;
-	display: flex;
-	align-items: center;
-	justify-content: space-around;
+  .breadcrumb {
+    display: inline-block;
+    margin: 15px;
+    width: 30px;
+    height: 30px;
+    border-radius: 50%;
+    opacity: 0.5;
+    background-color: #14d997;
+    &.active_route {
+      opacity: 1;
+    }
+  }
 }
 
-.breadcrumb {
-	width: 24px;
-	height: 24px;
-	border-radius: 50%;
-	opacity: 0.5;
-	background-color: #14D997;
-	&.active_route{
-		opacity: 1;
-	}
-}
\ No newline at end of file
+.create-campaign-subroute-wrapper {
+  padding: 32px;
+  .create-campaign-control-label {
+    display: inline;
+  }
+  .form-group {
+    div {
+      padding-left: 5px;
+      padding-right: 5px;
+    }
+    .readonly {
+      background-color: rgba(200, 200, 200, 0.9);
```

Comment:
Sass variable?

Reply:
chose to darken($mainwhite, 15%) instead